src/sourcekit-lsp/LanguageClientManager.ts

```diff
@@ -616,6 +620,36 @@ export class LanguageClientManager {
         this.languageClient = client;
         this.cancellationToken = new vscode.CancellationTokenSource();
 
+        this.languageClient.onRequest(
```

Comment:
Could we stick this in a separate function and also call it once the LSP server has started up next to where `activateLegacyInlayHints` is called.